aicsimageio/aics_image.py

```diff
@@ -112,12 +123,20 @@ def data(self):
         """
         Returns
         -------
-        returns a numpy.ndarray with dimension ordering "STCZYX"
+        Returns a numpy.ndarray with dimension ordering "STCZYX"
         """
         if self._data is None:
             reader_data = self._reader.data
+
+            # Handle delayed known dims reshape
+            if self._known_dims:
+                pass_dims = self._known_dims
+            else:
+                pass_dims = self.reader.dims
```

Comment:
Optionally: (this could also be done below on L139)
```suggestion
                pass_dims = self._known_dims or self.reader.dims
```

Reply:
Smart!

Reply:
It's enough to make me think it should be named override_dims instead of known_dims :)